R/task-purge.R

```diff
@@ -122,14 +126,16 @@ hipercow_purge <- function(task_ids = NULL,
   }
 
```

Comment:
Is it worth printing something indicating that the following lines are a work of fiction? We have (in the task logs printing) a pair of hoizontal rules that frame a block of output, here we might add something like that?

Reply:
Done with a cli_rule before and after with pointy arrows...